composer.json

```diff
@@ -15,11 +15,11 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.1",
-        "symfony/finder": "~2.1",
-        "symfony/yaml": "~2.3",
-        "symfony/console": "~2.3",
-        "symfony/process": "~2.3",
+        "symfony/framework-bundle": "~2.8||~3.0",
```

Comment:
I've ran a composer on it, but perhaps a single is correct? I guess they might both work in theory.